### DISCUSSION TITLE
CAPTCHA + ERROR 403

### DISCUSSION THREAD

**Discussion by Oubeoh** [jef/streetmerchant#1449] [General]
While I realize error 403 is due to IP ban, I can get it to work while running a VPN. With headless set to false, the VPN is not on, as it opens with a different chrome browser. The CAPTCHA is due to Amazon, and I am not sure how to work around that. Any help would be appreciated.

**Comment by absolutelyNoClue** [comment]
you have to reduce the frequency of checking on amazon to get rid of the CAPTCHA error. it worked for me.

**Reply by Ahland3r** [reply]
Is there a way to reduce just for Amazon or can we only adjust the Page Sleep for all sites with the general setting?

**Reply by absolutelyNoClue** [reply]
There is no config to set for a specific site only. It's either for all or none :-) Note that the CAPTCHA error will also pop up a lot of you are running in headless mode.

**Reply by WUZUP1998** [reply]
@absolutelyNoClue If you don't mind me asking, what did you put for your page sleep setting?

**Reply by absolutelyNoClue** [reply]
PAGE_SLEEP_MIN="50000"
PAGE_SLEEP_MAX="90000"
